lib/chat_app_web/live/home_live.ex

```diff
@@ -2,43 +2,43 @@ defmodule ChatAppWeb.HomeLive do
   use ChatAppWeb, :live_view
 
   def mount(_params, _session, socket) do
-    rooms = [
-      %{id: 1, name: "room1", description: "capybara numero uno"},
-      %{id: 2, name: "room2", description: "capybara numero dos"},
-      %{id: 3, name: "room3", description: "capybara numero tres"}
-    ]
+    rooms = ChatApp.Contexts.Rooms.list_rooms()
+    |> Enum.filter(fn room -> room.is_private == false end)
```

Comment:
the condition can be rewritten as `not room.is_private`.
Also, the formatting seems wrong, please `mix format`.